packages/desktop-client/src/hooks/usePayees.ts

```diff
@@ -7,17 +7,21 @@ import {
 
 import { useSelector, useDispatch } from '../redux';
 
+import { useInitialMount } from './useInitialMount';
+
 export function useCommonPayees() {
   const dispatch = useDispatch();
   const commonPayeesLoaded = useSelector(
     state => state.queries.commonPayeesLoaded,
   );
 
+  const isInitialMount = useInitialMount();
+
   useEffect(() => {
-    if (!commonPayeesLoaded) {
+    if (isInitialMount && !commonPayeesLoaded) {
```

Comment:
Question: I wonder if we should change this to refetch whenever the payees are unloaded